conf/fluo-deploy.props.example

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 [general]
+# type of cloud service
```

Comment:
would be nice to add the two possible values to the comment... 

Also thinking `ec2` would be nice instead of `aws-ec2` because its consistent with the name of the config section below